src/Runner.jl

```diff
@@ -46,7 +46,9 @@ and even simple programs will not compile without them.
 function generate_compiler_wrappers!(platform::Platform; bin_path::AbstractString,
                                      host_platform::Platform = Linux(:x86_64; libc=:musl),
                                      rust_platform::Platform = Linux(:x86_64; libc=:glibc),
-                                     compilers::Vector{Symbol} = [:c])
+                                     compilers::Vector{Symbol} = [:c],
+                                     allow_unsafe_flags::Bool = false,
```

Comment:
This allows for optionally allowing them, if really necessary, but we should propagate this all the way to `build_tarballs`.  Does it make sense?